### ISSUE TITLE
From which version is TabPFNRegressor available?

### ISSUE THREAD

**Issue by Zhangyh1432** [PriorLabs/TabPFN#248]
Hi TabPFN Team,

Thank you for your excellent work on TabPFN! I would like to ask from which version the TabPFNRegressor was introduced. I couldn't find a clear indication in the release notes or documentation.

It would be great if you could clarify this. Thanks in advance for your help!

Best regards

**Comment by Zhangyh1432** [comment]
I would also like to confirm if the latest version of TabPFN supports multi-output regression. If not, are there any plans to add this feature in the future?

**Comment by noahho** [comment]
Hi! It does not currently do so, but we will support this soon (The code is mostly ready in tabpfn-extensions repository). Are you looking for a point prediction or distribution for multi-output?
For your first question: It was release in version 2.0

**Comment by stlaoweng** [comment]
Hello, I have applied the TabPFN in my own datasets with the best performance compared to other ML models.
I am also looking forwards to the multi-output regression ( 1*n output ) , please inform me when the new version is available.

**Comment by ZiqiGuo98** [comment]
Hi, I am wondering if the multi-output regression is available now in the latest version of tabpfn. Thanks!

**Comment by noahho** [comment]
This is just being built here: https://github.com/PriorLabs/tabpfn-extensions/pull/207 and will be merged soon. You can already use this by installing from that branch

**Comment by noahho** [comment]
Multi-output regression works today with sklearn's wrapper, no TabPFN-specific class needed:

```python
from sklearn.multioutput import MultiOutputRegressor
from tabpfn import TabPFNRegressor

model = MultiOutputRegressor(TabPFNRegressor())
model.fit(X_train, y_train)   # y_train shape (n_samples, n_targets)
preds = model.predict(X_test)
```

This fits one TabPFN per target, so you get point predictions per output but no joint distribution across targets. The same pattern works with `MultiOutputClassifier` for multi-label. One gotcha there: `predict_proba` returns a list of per-target arrays, not a single array.

Runnable example with per-target R² and micro-ROC-AUC: [examples/multioutput/multioutput_prediction.py](https://github.com/PriorLabs/tabpfn-extensions/blob/main/examples/multioutput/multioutput_prediction.py) in tabpfn-extensions.

Closing since the pattern is documented now. Let us know if this didn't solve the question and we can reopen it.
